service/conceptAnnotation.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+// ConceptAnnotations models the annotations as it will be written on the queue
+type ConceptAnnotations struct {
+	UUID        string       `json:"uuid"`
+	Annotations []annotation `json:"annotations"`
+}
+
+type annotation struct {
+	Thing      thing        `json:"thing"`
+}
```

Comment:
why is this a Thing ? I'm assuming that isn't a neo Thing?

Reply:
Renamed to `Concept`